README.md

```diff
@@ -209,7 +216,7 @@ and improve:
 ![Architecture](docs/img/modin_architecture.png)
 
 Visit the [Documentation](https://modin.readthedocs.io/en/latest/developer/architecture.html) for
-more information, and checkout [the difference between Modin and Dask!](https://github.com/modin-project/modin/tree/master/docs/modin_vs_dask.md)
+more information, and checkout [the difference between Modin and Dask DataFrame!](https://github.com/modin-project/modin/tree/master/docs/modin_vs_dask.md).
```

Comment:
I propose moving this information about the difference between Modin and Dask DataFrame to the FAQ and removing it from the README. Thoughts?

Reply:
Should probably link this once that PR is merged: https://modin--3779.org.readthedocs.build/en/3779/getting_started/modin_vs_dask_vs_koalas.html

Reply:
Agreed, the README has a bit too much info now, so taking out this would be helpful.